src/main/java/nextstep/subway/auth/application/AuthService.java

```diff
@@ -6,34 +6,43 @@
 import nextstep.subway.auth.infrastructure.JwtTokenProvider;
 import nextstep.subway.member.domain.Member;
 import nextstep.subway.member.domain.MemberRepository;
+import nextstep.subway.member.exception.MemberExceptionCode;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 public class AuthService {
-    private MemberRepository memberRepository;
-    private JwtTokenProvider jwtTokenProvider;
+    private final MemberRepository memberRepository;
+    private final JwtTokenProvider jwtTokenProvider;
 
     public AuthService(MemberRepository memberRepository, JwtTokenProvider jwtTokenProvider) {
         this.memberRepository = memberRepository;
         this.jwtTokenProvider = jwtTokenProvider;
     }
 
+    @Transactional
```

Comment:
```suggestion
    @Transactional(readOnly = true)
```
`readOnly=true` 옵션을 붙이는건 어떨까요?

Reply:
상세한 답변 감사드립니다 ㅎㅎ 
근환님이 말씀해주신대로 Station이 삭제될때 Section에 대한 처리를 `StationService`에서 하는게 적절한지가
고민이 많이 되었었는데 답변이 많은 도움이 되었습니다! ㅎㅎ
구간에 대한 validation도 추가하지 않는 방향으로 진행하겠습니다 ㅎㅎ